### ISSUE TITLE
Godot crashes during import of GLTF2.0 with two and more morph shapes

### ISSUE THREAD

**Issue by Acvarium** [godotengine/godot#16124]
**Godot version:**
Godot 3.0 RC3

**OS/device including version:**
GNU/Linux lubuntu 16.04 x86_64 with Linux 4.6.4-040604-generic

**Issue description:**
Export file with one morph shape from blender to gltf 2.0, and it will import with no problem.
But if you add another shape, godot will crash during the import.

**Steps to reproduce:**
https://youtu.be/VkV-OfE2dus

**Minimal reproduction project:**
[godot3.0_morph_bug.blend.zip](https://github.com/godotengine/godot/files/1670655/godot3.0_morph_bug.blend.zip)

**Comment by pouleyKetchoupp** [comment]
Can still reproduce on master 60711e67386d8300878a46f3c37260db4f2b2c84

Error:
```
ERROR: CowData<float>::get: FATAL: Index p_index=4 out of size (size()=3)
   At: C:\dev\source\godot\core/cowdata.h:152
```
Backtrace:
```
[0] CowData<float>::get (C:\dev\source\godot\core\cowdata.h:152)
[1] Vector<float>::operator[] (C:\dev\source\godot\core\vector.h:85)
[2] EditorSceneImporterGLTF::_interpolate_track<float> (C:\dev\source\godot\editor\import\editor_scene_importer_gltf.cpp:2694)
[3] EditorSceneImporterGLTF::_import_animation (C:\dev\source\godot\editor\import\editor_scene_importer_gltf.cpp:2864)
[4] EditorSceneImporterGLTF::_generate_scene (C:\dev\source\godot\editor\import\editor_scene_importer_gltf.cpp:2930)
[5] EditorSceneImporterGLTF::import_scene (C:\dev\source\godot\editor\import\editor_scene_importer_gltf.cpp:3043)
[6] ResourceImporterScene::import (C:\dev\source\godot\editor\import\resource_importer_scene.cpp:1321)
[7] EditorFileSystem::_reimport_file (C:\dev\source\godot\editor\editor_file_system.cpp:1780)
[8] EditorFileSystem::reimport_files (C:\dev\source\godot\editor\editor_file_system.cpp:1979)
[9] EditorFileSystem::_update_scan_actions (C:\dev\source\godot\editor\editor_file_system.cpp:591)
[10] EditorFileSystem::_notification (C:\dev\source\godot\editor\editor_file_system.cpp:1162)
[11] EditorFileSystem::_notificationv (C:\dev\source\godot\editor\editor_file_system.h:108)
[12] Object::notification (C:\dev\source\godot\core\object.cpp:933)
[13] SceneTree::_notify_group_pause (C:\dev\source\godot\scene\main\scene_tree.cpp:983)
[14] SceneTree::idle (C:\dev\source\godot\scene\main\scene_tree.cpp:527)
[15] Main::iteration (C:\dev\source\godot\main\main.cpp:1980)
[16] OS_Windows::run (C:\dev\source\godot\platform\windows\os_windows.cpp:3142)
[17] widechar_main (C:\dev\source\godot\platform\windows\godot_windows.cpp:162)
[18] _main (C:\dev\source\godot\platform\windows\godot_windows.cpp:184)
[19] main (C:\dev\source\godot\platform\windows\godot_windows.cpp:196)
[20] __scrt_common_main_seh (d:\agent\_work\3\s\src\vctools\crt\vcstartup\src\startup\exe_common.inl:288)
[21] BaseThreadInitThunk
```

**Comment by pouleyKetchoupp** [comment]
From what I've seen this is an issue from Blender's gltf exporter itself.
Whatever the number of morph targets, the number of animation values is always 6, instead of `number of morph targets` * `number of frames`.

*gltf 2.0 Specifications:*
https://github.com/KhronosGroup/glTF/tree/master/specification/2.0
>A Morph Target animation frame is defined by a sequence of scalars of length equal to the number of targets in the animated Morph Target. These scalar sequences must lie end-to-end as a single stream in the output accessor, whose final size will equal the number of Morph Targets times the number of animation frames.

*Ticket on Blender exporter* 
https://github.com/KhronosGroup/glTF-Blender-IO/issues/641

It might get fixed in the next versions of Blender, but from the tests I've made with the fix proposed in this ticket, the exported data was still invalid.

**Comment by akien-mga** [comment]
IIUC this can be considered fixed by #33794 and Blender 2.81?

**Comment by pouleyKetchoupp** [comment]
Yes, that's correct.
